### PR TITLE
Name Change for Windows Patching

### PR DESCRIPTION
**Pull request by aimeemudd**
Changing the name of the Windows Patching Blueprint and screenshots

### CODE REVIEW

**Comment by osbornm** [comment]
Please merge master into your branch... https://github.com/CenturyLinkCloud/PublicKB#merging-an-upstream-repository-into-your-fork

**Comment by aimeemudd** [comment]
When following the given steps I am running into a permissions error on merging.

**Comment by osbornm** [comment]
that means you are trying to mergeaimeemudd/master into clc/master, thats not what you want. you want to merge clc/master into aimeemudd/master

**Comment by aimeemudd** [comment]
ok I think I finally did this right!

File: Servers/patching-as-a-service.md [hunk 1]
```diff
@@ -71,7 +71,11 @@ For any method you choose, the following table will assist you with data needed
 
 **Operating Systems** | **Blueprint Name** | **Script Package Name** | **Package ID**
  --- | --- | --- | ---
+<<<<<<< HEAD
+ Windows 2012 and 2012R2 | Auto Patching Windows 2012 | Auto Patching Windows 2012 | 94bd395f-9b54-4693-94ae-4f3aa93fc239
+=======
  Windows 2012 and 2012R2 | Windows Update Scripts | Windows Update Scripts | 94bd395f-9b54-4693-94ae-4f3aa93fc239
+>>>>>>> 4817eb59752647ca87c0be2a83cef557fa059705
```
Comment (osbornm) [inline Servers/patching-as-a-service.md]: your merge did not go right these are merge conflicts.

**Comment by osbornm** [comment]
sorry but no :(

**Comment by aimeemudd** [comment]
fixed.

**Pull request merged**